include/qstls.hpp

```diff
@@ -75,8 +71,13 @@ class Qstls : public Stls {
 		    vecUtil::Vector3D &adrFixed_,
 		    double &Theta,
 		    int &nl) const;
+
 public:
 
+  vecUtil::Vector3D adrFixed;

```

Comment:
I don't like that this is public. It would be better to  make it private again and have a getter method. Something along this line

```
const vecUtil::Vector3D& getAdrFixed() const { return adrFixed; }